tests/collector.py

```diff
@@ -24,3 +27,9 @@ def test_parse_indexes(self):
     self.assertEqual({'l': 'octet'},
         parse_indexes((3, 65, 32, 255), [{'labelname': 'l', 'type': 'OctetString'}],
                       [{'labels': ['l'], 'labelname': 'l', 'oid': '1'}], oids))
+    self.assertEqual({'l': '10.181.1.30'},
```

Comment:
You should also test this without the lookup, so we know the rendering into the label value is correct